sdks/python/apache_beam/metrics/execution.py

```diff
@@ -34,11 +34,12 @@
 
 from apache_beam.metrics.cells import CounterCell, DistributionCell
 
+__all__ = [
+    'MetricKey', 'MetricResult', 'MetricsEnvironment', 'MetricsContainer']
```

Comment:
They are not used, and the file already has an internal comment, I will revert this.